docs/indepth/installation.rst

```diff
@@ -437,70 +477,44 @@ The optional ``port`` variable is the port on the control node to connect to:
    "control-service":
       "hostname": "${CONTROL_NODE}"
       "port": 4524
-  	"dataset":
-  	  "backend": "zfs"
-
-.. task:: enable_flocker_agent ubuntu-14.04 ${CONTROL_NODE}
-   :prompt: [root@agent-node]#
-
-What to do next
----------------
-
-You have now installed ``clusterhq-flocker-node`` and created a ZFS for it.
-
-Next you may want to perform the steps in :doc:`the tutorial <./tutorial/moving-applications>`, to ensure that your nodes are correctly configured.
-Replace the IP addresses in the ``deployment.yaml`` files with the IP address of your own nodes.
-Keep in mind that the tutorial was designed with local virtual machines in mind, and results in an insecure environment.
-
-
-ZFS Backend Configuration
--------------------------
-
-The ZFS backend requires ZFS to be installed.
-
-
-Installing ZFS on CentOS 7
-..........................
-
-Installing ZFS requires the kernel development headers for the running kernel.
-Since CentOS doesn't provide easy access to old package versions,
-the easiest way to get appropriate headers is to upgrade the kernel and install the headers.
+   "dataset":
+      "backend": "zfs"
+      "pool": "flocker"
```

Comment:
I've just done some testing and I find that the key name is `name` and that there are some other required variables.

```
dataset:
    backend: zfs
    name: 'flocker'
    mount_root: '/tmp/flocker-mount-root'
    volume_config_path: ''

```

ie keys for all the arguments that get supplied to `_zfs_storagepool`